server/routes/analyticsRouter.ts

```diff
@@ -138,14 +138,22 @@ export default function routes(router: Router): Router {
     const prisonersByEthnicity = await transformAnalyticsError(
       analyticsService.getIncentiveLevelsByProtectedCharacteristic(activeCaseLoad, ProtectedCharacteristic.Ethnicity)
     )
-    const prisonersInAgeGroups = await transformAnalyticsError(
+    const prisonersByAgeGroup = await transformAnalyticsError(
       analyticsService.getIncentiveLevelsByProtectedCharacteristic(activeCaseLoad, ProtectedCharacteristic.AgeGroup)
     )
+    const prisonersByReligion = await transformAnalyticsError(
```

Comment:
If we wanted to reduce the length of this function I think you could refactor it to use
`const [prisonersByAgeGroup, prisonersByReligion, prisonersByDisability] = await Promise.all[(<each call to analyticsService>)]`

Reply:
done!